optuna/testing/distributions.py

```diff
@@ -15,3 +16,6 @@ def _contains(self, param_value_in_internal_repr: float) -> bool:
     def _asdict(self) -> Dict:
 
         return {}
+
+    def to_internal_repr(self, param_value_in_external_repr: Any) -> float:
+        return param_value_in_external_repr
```

Comment:
Doesn't `mypy` causes an error on this since it implicitly cast the `Any` value to `float`? It might be good to cast explicitly the value to `float`.

Reply:
Thank you for your comment. I add cast to float in next commit.